docker-compose.yml.dist

```diff
@@ -2,6 +2,7 @@ mail:
   image: tvial/docker-mailserver
   hostname: mail
   domainname: domain.com
+  privileged: true
```

Comment:
To ban ip addresses fail2ban uses iptables. Running in privileged mode will allow to run without the permission denied message: "permission denied (you must be root)"